packages/lesswrong/components/posts/PostsByVote.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useMulti } from '../../lib/crud/withMulti';
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+
+const PostsByVote = ({postIds, year}:{postIds: Array<string>, year: number}) => {
+    const { PostsItem2, ErrorBoundary } = Components
```

Comment:
This file is a combination of 4 and 2 -spaced indent. The rest of the codebase is on 2, so to support copy-pasting, please use 2.